api/v1/clusterextension_types.go

```diff
@@ -138,8 +138,8 @@ type ClusterExtensionInstallConfig struct {
 	Preflight *PreflightConfig `json:"preflight,omitempty"`
 }
 
-// CatalogSource defines the attributes used to identify and filter content from a catalog.
-type CatalogSource struct {
+// CatalogFilter defines the attributes used to identify and filter content from a catalog.
+type CatalogFilter struct {
```

Comment:
Do we need to change the name of the type?
Would not it be a real breaking change in the API/end users CRD?

Reply:
Yes we do. Otherwise, there is a conflict with catalogd CatalogSource golang data structure. The first line of the commit comment says:

>This required renaming op-con's CatalogSource to CatalogFilter to avoid a name conflict with catd's CatalogSource

There is no change to the CRD, as the CRD does not reference golang data structures. You will notice that the CRDs in `config/...` are not modified.